resources/services/athena/data_catalogs.go

```diff
@@ -339,6 +339,7 @@ func dataCatalogDetail(ctx context.Context, meta schema.ClientMeta, resultsChan
 			return
 		}
 		errorChan <- diag.WrapError(err)
+		return
```

Comment:
strange that this makes cloudquery crash.

Reply:
I can recreate the issue locally